### PR TITLE
Pqa tool

### PR DESCRIPTION
**Pull request by SamCox822**

### CODE REVIEW

File: mdagent/tools/base_tools/util_tools/search_tools.py [hunk 1]
```diff
@@ -1,26 +1,75 @@
...
+    return answer
 
 
 class Scholar2ResultLLM(BaseTool):
     name = "LiteratureSearch"
-    description = """Input a specific question,
-                returns an answer from literature search."""
+    description = (
+        "Useful to answer questions that require technical "
+        "knowledge. Ask a specific question."
```
Comment (Jgmedina95) [inline mdagent/tools/base_tools/util_tools/search_tools.py:62]: maybe add a couple examples of "Specific questions"

File: mdagent/utils/general_utils.py
Comment (Jgmedina95) [inline mdagent/utils/general_utils.py:1]: where is this being used?

**Review by Jgmedina95 — COMMENTED** [review]
Overall, i like the idea that we don't need a pqa API key anymore. My only question, not even a concern, is where is the function find_file_path used?

File: mdagent/utils/general_utils.py
Comment (SamCox822) [reply mdagent/utils/general_utils.py:1]: thanks for point this out - its an artifact of something I was doing for another PR - I removed it

**Comment by SamCox822** [comment]
> Overall, i like the idea that we don't need a pqa API key anymore. My only question, not even a concern, is where is the function find_file_path used?

fixed this - thank you!

**Review by Jgmedina95 — APPROVED** [review]
LGTM

**Pull request merged**